packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -181,7 +181,8 @@
                 }
               }
             ]
-          }
+          },
+          "guidancePanel": { "$ref": "#/definitions/guidancePanel" }
```

Comment:
Also sorry to make work for you but I merged in my nav panel change whcih added the `appDetails` object to this schema. Could you add a line to that part of the schema that is just like this one, so the `appDetails` has a `guidancePanel`?

Reply:
No worries at all! Updated it :))